src/vma/sock/sockinfo_udp.cpp

```diff
@@ -1756,6 +1756,7 @@ int sockinfo_udp::rx_request_notification(uint64_t poll_sn)
 ssize_t sockinfo_udp::tx(const tx_call_t call_type, const iovec* p_iov, const ssize_t sz_iov,
 		     const int __flags /*=0*/, const struct sockaddr *__dst /*=NULL*/, const socklen_t __dstlen /*=0*/)
 {
+	static bool use_hyperv = (mce_sys_var::HYPER_MSHV == safe_mce_sys().hypervisor);
```

Comment:
Please replace with class paramater.
please rename to ms_hyperv

Reply:
done